__tests__/app.test.js

```diff
@@ -100,4 +100,49 @@ describe('/api/articles', () => {
             })
         })
     })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 sends an array of comments for the given article id', () => {
+        return request(app).get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            const { comments } = body
+
+            expect(comments).toBeSortedBy('created_at', {descending: true, coerce: true})
+            expect(comments.length).toBe(11)
+            
+            comments.forEach((comment) => {
+                expect(comment).toHaveProperty('comment_id', expect.any(Number))
+                expect(comment).toHaveProperty('votes', expect.any(Number))
+                expect(comment).toHaveProperty('created_at', expect.any(String))
+                expect(comment).toHaveProperty('author', expect.any(String))
+                expect(comment).toHaveProperty('body', expect.any(String))
+                expect(comment).toHaveProperty('article_id', expect.any(Number))

```

Comment:
We should be more specific on the article_id value as we know what this will be based on the request :)